clap_derive/tests/ui/tuple_struct.stderr

```diff
@@ -3,6 +3,8 @@ error: `#[derive(Clap)]` only supports non-tuple structs and enums
    |
 11 | #[derive(Clap, Debug)]
    |          ^^^^
+   |
+   = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
Are you sure we won't have any issues on 1.40?

Reply:
We don't run trybuild on non-latest rustc: https://github.com/clap-rs/clap/blob/99b1b768c25c1f317174e0b42e6740884c620f44/clap_derive/tests/macro-errors.rs#L8

But when our MSRV will hit 1.42, we will have a problem, as we had in https://github.com/clap-rs/clap/pull/1745 (bumped MSRV from 1.36 to 1.40). I don't think this is a big burden, but I'll ready a separate PR for more bullet-proof checking system.

Reply:
Hey @CreepySkeleton I faced an error today specifically in `clap_derive/tests/ui/tuple_struct.stderr` while running tests as part of #1840 . It's spitting older stderr output only for `tuple_struct.rs` test in `clap_derive/tests/ui`.

rustc version: `rustc 1.42.0 (b8cedc004 2020-03-09)`. 

Found via `git blame` that this commit added these two extra lines in `tuple_struct.stderr`.
My local fork is rebased with latest commits as of 2020-04-26, but I still get the error. 

Any idea if my PR (#1840) has changes that could be leading to the older stderr output and not the new expected output from this (#1857) PR?

Reply:
Basically this UI tests run only on 1.43. We are not testing on 1.42 anywhere on CI, so please go ahead and make a PR, your PR shouldn't fail CI.